smtp-server/src/lib.rs

```diff
@@ -676,6 +727,8 @@ where
                                 }
                                 reader.complete();
                                 unhandled = reader.get_unhandled().unwrap();
+                                // TODO: in the case of LMTP, must we send multiple error replies?
+                                drop(decision_stream); // to free &mut conn_meta
```

Comment:
Did the code not compile without this? It seemed to me that since NLL that kind of code should work even without this line. But then maybe I'm mistaking with Polonius?

Reply:
:+1: 

(Polonius is basically https://docs.rs/polonius-the-crab/0.2.1/polonius_the_crab/ but at the compiler level)